jerry-core/ecma/builtin-objects/ecma-builtins.h

```diff
@@ -23,13 +23,26 @@
  */
 typedef enum
 {
+#define BUILTIN(a, b, c, d, e)
+#define BUILTIN_CALLABLE(builtin_id, \
+                object_type, \
+                object_prototype_builtin_id, \
+                is_extensible, \
+                lowercase_name) \
+  builtin_id,
```

Comment:
Is this separation required somewhere? That is do we expect the order of the builtin enums somewhere?

Reply:
Yes, it is necessary to be in that order.